examples/fabric/reinforcement_learning/train_torch.py

```diff
@@ -204,9 +202,9 @@ def main(args: argparse.Namespace):
 
             # Single environment step
             next_obs, reward, done, truncated, info = envs.step(action.cpu().numpy())
-            done = np.logical_or(done, truncated)
+            done = torch.logical_or(Tensor(done), Tensor(truncated))
```

Comment:
```suggestion
            done = torch.logical_or(torch.tensor(done), torch.tensor(truncated))
```

Reply:
Thanks!  Didn't know about the tensor/Tensor convention.  
Made two commits, first for the suggestions you gave and then there's one or two others in places I hadn't previously written, so we can remove the `from torch import Tensor` line too.